src/ontology/mondo-edit.obo

```diff
@@ -287721,10 +287721,13 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0013999
-name: optic nerve edema-splenomegaly syndrome
-def: "Optic nerve edema-splenomegaly syndrome is a rare presumably genetic disorder characterized by idiopathic massive splenomegaly with pancytopenia and childhood-onset chronic optic nerve edema with slowly progressive vision loss. Additional reported features include anhidrosis, urticaria and headaches." [Orphanet:313800]
+name: retinal dystrophy, optic nerve edema, splenomegaly, anhidrosis, and migraine headache syndrome
```

Comment:
@nicolevasilevsky in the issue, I don't see "migraine" in the requested label. 
Is this a mistake, or do you have evidence to include this name?

Reply:
this is the OMIM label.